libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/route/line/RouteLineTrimOffset.kt

```diff
@@ -0,0 +1,9 @@
+package com.mapbox.navigation.ui.maps.route.line
```

Comment:
This should be in the model package next to the other classes it's related to.

Reply:
Addressed and re-pushed, let me know if looks good now 🙏

Reply:
merge it